### DISCUSSION TITLE
List of Apps?

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#1088] [General]
Is there a list of Apps compatible with Miyoo out there somewhere? I'm looking for a overclocking app which I think exists but I can't find it for example

**Comment by thomasklemenc** [comment]
[here is a list](https://github.com/OnionUI/Onion/wiki/Third-Party-Apps)

**Reply by dreams897** [reply]
Thank you that is very helpful, however I'm still not seeing overclocking tools and I can't help but feel like some other apps are missing

**Reply by XK9274** [reply]
The overclocking tool probably won't get added but you can get it in RGH discord.

What apps do you feel are missing?
